src/utils/url.js

```diff
@@ -1,12 +1,18 @@
-import { environment } from "../environments/environment";
-
 const URL = {
     login: () => {
         return `login`;
     },
 
+    signup: () => {
+        return `signup`;
+    },
+
     users: () => {
         return `users`;
+    },
+
+    user: (id) => {
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/abdulqhayum47/lets-react/pullRequest?prid=9741124)
```suggestion
  user: (id) => {
```